features/upgrade/sdn/policy-upgrade.feature

```diff
@@ -0,0 +1,47 @@
+Feature: SDN compoment upgrade testing
+  # @author zzhao@redhat.com
+  @admin
+  @upgrade-prepare
+  Scenario: Check the networkpolicy works well after upgrade - prepare
+    Given I switch to cluster admin pseudo user		
+    When I run the :new_project client command with:
+      | project_name | policy-upgrade |		
+    Then the step should succeed
+    When I use the "policy-upgrade" project      
+    And I run the :create client command with:
+      | f | <%= ENV['BUSHSLICER_HOME'] %>/testdata/networking/list_for_pods.json |
+    Then the step should succeed
+    Given 2 pods become ready with labels:
+      | name=test-pods |
+    And evaluation of `pod(1).ip` is stored in the :p1pod2ip clipboard
+    And evaluation of `pod(0).name` is stored in the :p1pod1 clipboard
```

Comment:
What's the p1 mean ? Is it a must ?

Reply:
no more meaning, just name it.

Reply:
Then please update it with simple names, e.g pod1_ip, pod2_name

Reply:
I will not update it since all my cases using this kind name.

Reply:
@zhaozhanqi I agree with @liangxia in that it'll make it easier for others to follow if the variable names are assign in a clearer fashion.  Please update the scenario according to his suggestion.  Thanks!

Reply:
ah, ok.  Thought what kind of things block this to be merged so long time. 
anyway, I updated them as expected.  please help check, thanks.